protocol/storage/miner_test.go

```diff
@@ -100,7 +108,8 @@ func TestReceiveStorageProposal(t *testing.T) {
 		assert := assert.New(t)
 		require := require.New(t)
 
-		porcelainAPI, miner, proposal := newMinerTestSetup()
+		porcelainAPI := newMinerTestPorcelain(require)
+		miner, proposal := newMinerTestSetup(porcelainAPI, VoucherInterval, defaultAmountInc)
```

Comment:
Thank you, this flexibility is an improvement. These "common" setups could be factored into a `defaultMinerTestSetup` returning the triplet, just for conciseness: your call.